lib/rouge/lexers/http.rb

```diff
@@ -7,8 +7,8 @@ class HTTP < RegexLexer
       title "HTTP"
       desc 'http requests and responses'
 
-      def self.methods
```

Comment:
Ah, so that's an interesting one. It turns out that the existing `def self.analyze_text; ...; end` methods in lexers are getting inherited - so for example QML (which is an extension of javascript) has been triggered when the text starts with `#!/usr/bin/env node`. To avoid this, we're checking whether the class implements its own `analyze_text` method using `#methods`... but this broke on the HTTP lexer because we're overrideing `#methods` on the class.